src/bit-systems/media-loading.ts

```diff
@@ -17,6 +18,54 @@ import { loadVideo } from "../utils/load-video";
 import { MediaType, mediaTypeName, resolveMediaInfo } from "../utils/media-utils";
 import { EntityID } from "../utils/networking-types";
 
+export enum MediaTypeE {
+  VIDEO,
+  GLTF,
+  PDF,
+  IMAGE,
+  HTML
+}
+
+function contentTypeForMediaInfo(mediaInfo: MediaInfo) {
+  const { contentType, accessibleUrl } = mediaInfo;
+  if (
+    contentType.startsWith("video/") ||
+    contentType.startsWith("audio/") ||
+    contentType.startsWith("application/dash") ||
+    AFRAME.utils.material.isHLS(accessibleUrl, contentType)
+  ) {
+    return MediaTypeE.VIDEO;
+  } else if (
+    contentType.includes("application/octet-stream") ||
+    contentType.includes("x-zip-compressed") ||
+    contentType.startsWith("model/gltf")
+  ) {
+    return MediaTypeE.GLTF;
+  } else if (contentType.startsWith("image/")) {
+    return MediaTypeE.IMAGE;
+  } else if (contentType.startsWith("application/pdf")) {
+    return MediaTypeE.PDF;
+  } else if (contentType.startsWith("text/html")) {
+    return MediaTypeE.HTML;
+  }
+}
+
+function addMediaPhysicsShape(world: HubsWorld, eid: number, mediaInfo: MediaInfo) {
+  const mediaType = contentTypeForMediaInfo(mediaInfo);
```

Comment:
Yeah, I think this was part of a first iteration where I needed this info somewhere else and then I left it thinking that it would be useful to expose that `MediaInfo` outside this system but I agree that it's better to keep it simple.